source/ui/VESPERS/VESPERSEXAFSScanConfigurationView.cpp

```diff
@@ -430,6 +430,14 @@ void VESPERSEXAFSScanConfigurationView::setScanPosition()
 		savedYPosition_->setText(QString("Z: %1 mm").arg(y, 0, 'g', 3));
 	}
 
+	else if (motor == (VESPERS::BigBeamX | VESPERS::BigBeamZ)){
+
+		x = VESPERSBeamline::vespers()->bigBeamMotorGroupObject()->horizontalAxis()->translationMotor()->value();
+		y = VESPERSBeamline::vespers()->bigBeamMotorGroupObject()->verticalAxis()->translationMotor()->value();
+		savedXPosition_->setText(QString("X: %1 mm").arg(x, 0, 'g', 3));
+		savedYPosition_->setText(QString("Z: %1 mm").arg(y, 0, 'g', 3));
```

Comment:
Here again the y/z mismatch. Just a quirk of using different coordinate systems? Or doesn't mean what I think it means?